src/commands/create.js

```diff
@@ -68,9 +68,9 @@ ${chalk.green('=> To get started:')}
 
     cd ${answers.name}
 
-    ${isYarn ? 'yarn' : 'npm run'} start ${chalk.green('- Start the development server')}
-    ${isYarn ? 'yarn' : 'npm run'} build ${chalk.green('- Build for production')}
-    ${isYarn ? 'yarn' : 'npm run'} serve ${chalk.green('- Test a production build locally')}
+    ${isYarn ? chalk.hex('#2c8ebb')('yarn') : chalk.hex('#CB3837')('npm run')} start ${chalk.green('- Start the development server')}
+    ${isYarn ? chalk.hex('#2c8ebb')('yarn') : chalk.hex('#CB3837')('npm run')} build ${chalk.green('- Build for production')}
+    ${isYarn ? chalk.hex('#2c8ebb')('yarn') : chalk.hex('#CB3837')('npm run')} serve ${chalk.green('- Test a production build locally')}
```

Comment:
We should really extract these colors out into the utils or something.  
eg.
```
// Utils.js
export const Colors = {
  yarn: '...',
  npm: '...'
}
```

Heck, we should even Proxy our calls to `chalk` through our own `Chalk.js` file, so we can just use `Chalk.yarn(...)`

Reply:
That could actually be a lot cleaner method of making Chalk calls 🤔. 
I'll work on splitting that out.